jupyterhub/metrics.py

```diff
@@ -3,9 +3,9 @@
 
 Read https://prometheus.io/docs/practices/naming/ for naming
 conventions for metrics & labels. We generally prefer naming them
-`<noun>_<verb>_<type_suffix>`. So a histogram that's tracking
+`jupyterhub_<noun>_<verb>_<type_suffix>`. So a histogram that's tracking
 the duration (in seconds) of servers spawning would be called
-SERVER_SPAWN_DURATION_SECONDS.
+jupyterhub_SERVER_SPAWN_DURATION_SECONDS.
```

Comment:
the all caps names are the object names used internally, but I think this is about the metric names, so:

```suggestion
jupyterhub_server_spawn_duration_seconds.
```

Reply:
Can we also add in the module docstring:

```rst
.. versionchanged:: 1.3

    added ``jupyterhub_`` prefix to all metric names
```